### DISCUSSION TITLE
Feature request: pre selectable orders for Handyman/Mechanic

### DISCUSSION THREAD

**Discussion by dev-aphex** [OpenRCT2/OpenRCT2#15769] [Feature requests]
and/or clickable orders at Staff-Window. For example if you want to deny mowing for every Handyman at once without switching trough everyone one for one.

**Comment by Gymnasiast** [comment]
Handyman do not mow gross by default anyway.

Clickable orders in the Staff List window does sound like a nice idea though.

**Comment by blackhand1001** [comment]
I'd like to see an option to disable grass growing in general to be honest. Its a game mechanic that never really was balanced properly to begin with. it grows too fast and takes too long to mow.  Probably why they removed it in RCT3.
